expand.cpp

```diff
@@ -204,10 +204,9 @@ namespace Sass {
   {
     String* old_p = d->property();
     String* new_p = static_cast<String*>(old_p->perform(eval->with(env, backtrace)));
-    Expression* value = d->value()->perform(eval->with(env, backtrace));
-
+    Selector* p = selector_stack.size() >= 1 ? 0 : selector_stack.back();
```

Comment:
ah, It isn't supposed to be.  I was using it in the next line and forgot to put it back in.  Committing in a moment...